dashboard/src/components/AppView/AccessURLTable/AccessURLTable.test.tsx

```diff
@@ -141,8 +164,29 @@ context("when the app contain services and ingresses", () => {
       } as IIngressSpec,
     } as IResource;
     const ingresses = [{ isFetching: false, item: ingress }];
-    const wrapper = shallow(<AccessURLTable services={services} ingresses={ingresses} />);
+    const wrapper = shallow(
+      <AccessURLTable services={services} ingresses={ingresses} fetchIngresses={jest.fn()} />,
+    );
     expect(wrapper.find(AccessURLItem)).toExist();
     expect(wrapper).toMatchSnapshot();
   });
 });
+
+context("when the contains resources with errors", () => {
+  it("displays the error", () => {
+    const services = [{ isFetching: false, error: new Error("could not find Service") }];
+    const ingresses = [{ isFetching: false, error: new Error("could not find Ingress") }];
+    const wrapper = shallow(
+      <AccessURLTable services={services} ingresses={ingresses} fetchIngresses={jest.fn()} />,
+    );
+    expect(wrapper.find(AccessURLItem)).not.toExist();
+    expect(wrapper.find("table").text()).toContain("could not find Ingress");
+
+    // The Service error is not shown, as it is filtered out because without the
+    // resource we can't determine whether it is a public LoadBalancer Service
+    // or not. The Service error will be shown in the Services table anyway.
+    expect(wrapper.find("table").text()).not.toContain("could not find Service");
```

Comment:
do we need this `assert` then? It seems that we are ensuring that a side effect exists. I would just remove this assert and leave the comment which explains the situation very well.

Reply:
@andresmgot good point, you're right - I'll remove it!